hermes-common/src/main/java/pl/allegro/tech/hermes/common/kafka/KafkaNamesMapper.java

```diff
@@ -40,4 +44,15 @@ private String namespaced(String name) {
         return namespace + SEPARATOR + name;
     }
 
+    private String topicNameSuffix(Topic.ContentType contentType) {
```

Comment:
we already have a few places like this, we should think about refactoring in a near future to make all type-related switches and ifs easy to find and extend